docs/manual/troubleshooting.tex

```diff
@@ -968,6 +968,44 @@ \subsection{Build the Checker Framework Manual (this document)\label{building-ma
 \end{enumerate}
 
 
+\subsection{Configure Eclipse to Edit Checker Framework\label{building-eclipse}}
+\begin{enumerate}
+\item
+Install Eclipse from the Eclipse official website.
+
+\item
+Once installation is complete, run Eclispe and set the working directory to jsr308.
```

Comment:
Should the `jsr308` be `$JSR308`?
Use `\<>` for in-line terminal font, throughout your changes.